clang/test/SemaCXX/ctad-copy-init-list-narrowing.cpp

```diff
@@ -0,0 +1,65 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -Wno-unused-value -std=c++20 %s
```

Comment:
nit: please format the test file.

Reply:
Applied clang format, nothing has really changed. I have changed the formatting by hand at some places.

Reply:
yeah, that's expected, we disable the clang-format for all lit test files (see the `clang/test/.clang-format`).